zerver/tests/test_users.py

```diff
@@ -1595,6 +1627,28 @@ def test_get_user_profile(self) -> None:
         self.assertEqual(result['user']['email'], bot.email)
         self.assertTrue(result['user']['is_bot'])
 
+        # Tests the GET /users/{email} api endpoint.
+        user = self.example_user('hamlet')
+        result = ujson.loads(self.client_get(f'/json/users/{user.email}').content)
```

Comment:
We removed `ujson` from the codebase in favor of `orjson`; please rebase and fix.

Also, we should use our standard `assert_json_success` pattern for these API requests, rather than wrapping a JSON load around the `result.content` object in a one-liner.  Maybe fix the surrounding tests in a prep commit?

Reply:
Yeah,  I have rebased my local branch, but didn't push yet.